Rakefile

```diff
@@ -36,4 +36,22 @@ Rake::TestTask.new do |t|
   t.verbose = true
 end
 
-task default: [:test, :rubocop]
+task default: [:test, 'rubocop:auto_correct']
```

Comment:
:( So In my opinion :-1: to this, I think it is a little weird you run `$ rake` and get your code modifed after that.